usaspending_api/disaster/v2/views/agency/spending.py

```diff
@@ -69,7 +69,11 @@ def post(self, request):
             results = self.total_queryset
 
         results = list(results.order_by(*self.pagination.robust_order_by_fields))
-
+        for item in results:  # we're checking for items that do not have an agency profile page
+            if item.get("link") is not None:
+                if not item["link"]:
```

Comment:
Will `item["id"] = None` ever be reached? The `not item["link"]` seems like it would always return False since you only reach this block when "item" has a "link".

Or is this checking for empty string or other things that evaluate to False?

Reply:
`item["link"]` is a boolean value, if it's false, we assign the id to `None`.

Reply:
Gotcha, that makes more sense. Thanks!